lumber-generate.js

```diff
@@ -10,6 +10,7 @@ const logger = require('./services/logger');
 program
   .description('Generate a backend application with an ORM/ODM configured.')
   .option('-c, --connection-url <connectionUrl>', 'Enter the database credentials with a connection URL')
+  .option('-H, --application-host <applicationHost>', 'Hostname of your admin backend app')
   .option('-p, --application-port <applicationPort>', 'Port of your admin backend app')
```

Comment:
[Wording] Also a detail but I would replace "app" by "application" in the option descriptions.
We tend not to "truncate" our variable names, so I thing we should do the same for the wording.
_Even though I agree that "app" is almost a real word now :)_